src/main/java/com/imagelab/App.java

```diff
@@ -0,0 +1,39 @@
+package com.imagelab;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+/**
+ * JavaFX App
+ */
```

Comment:
Doc comments should conclude with a period. Let's add this as one of our checkstyles rules.